lib/tools/system-commands.js

```diff
@@ -72,9 +74,44 @@ systemCallMethods.connect = async function connect (opts = {}) {
     }
   }
 
+  let port = DEFAULT_COMPANION_PORT;
```

Comment:
does it work if the companion already running?

Reply:
It works insofar as it kills the running companion.